style.css

```diff
@@ -695,3 +690,389 @@ footer {
+
+.hard-fork-latest {
+  position: relative;
+  top: 19.5rem;
+  padding-top: 1%;
+  background-color: rgb(248, 211, 248);
+  height: 5rem;
+  color: #f42;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  bottom: 7rem;
+}
+
+.hard-fork-image-size {
+  width: 100%;
+}
+
+.hard-fork-image {
+  border: 1px solid green;
+}
+
+.hard-fork-latest-paragraph {
+  color: red;
+}
+
+.hard-fork-2 {
+  position: relative;
+  top: 6rem;
+  height: auto;
+}
+
+.hard-fork-2 ul li ::before {
+  content: "\2022";
+  color: red;
+  margin-right: 1rem;
+}
+
+.hard-fork-2 ul {
+  list-style: none;
+}
+
+.hard-fork-2 ul li a:hover {
+  color: black;
+}
+
+.hard-fork-image-22 {
+  width: 30%;
+  float: right;
+}
+
+/** latest-deals-Section-desktop **/
+.latest-deals-header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: baseline;
+  padding-bottom: 2%;
+  padding-top: 4%;
+}
+
+.latest-deals-header a h1 {
+  color: #f42;
+  font-size: 200%;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+.latest-deals-header a h1:hover {
+  color: black;
+  font-size: 200%;
+}
+
+.latest-deals-container {
+  display: -ms-grid;
+  display: grid;
+  grid-template: 1fr 1fr / repeat(4, 24%);
+  grid-column-gap: 2%;
+  grid-row-gap: 2%;
+  width: 100%;
+  height: auto;
+  padding-bottom: 5%;
+  z-index: 1;
+}
+
+.latest-deals-image-size {
+  clip-path: polygon(66% 0, 100% 34%, 100% 100%, 0 100%, 0 0);
+  width: 100%;
+}
+
+.latest-deals-1 {
+  background: white;
+}
+
+.latest-deals-container-image {
+  position: relative;
+  background: yellow;
+}
+
+.latest-deals-container-image-yellow {
+  font-size: 75%;
+  font-weight: bold;
+  position: absolute;
+  left: 77%;
+  top: 9%;
+  transform: rotate(40deg);
+  background: yellow;
+}
+
+.latest-deals-text {
+  padding-top: 5%;
+}
+
+.latest-deals-text a h4 {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  color: #000;
+}
+
+.latest-deals-text a h4:hover {
+  color: #f42;
+}
+
+.latest-deals-text a span {
+  color: #555;
+  text-decoration: line-through;
+  font-size: 80%;
+}
+
+@media screen (min-width: 767px) and (max-width: 1024px) {
+  /** latest-news-ipad **/
+  .latest-news-grid {
+    display: -ms-grid;
+    display: grid;
+    grid-template: 1fr 1fr 1fr / repeat(3, 33%);
+    grid-column-gap: 2%;
+    grid-row-gap: 2%;
+    width: 100%;
+    height: auto;
+  }
+
+  /** hard-fork-ipad **/
+  .hard-fork {
+    display: -ms-grid;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-row-gap: 10%;
+    grid-column-gap: 2%;
+    width: 100%;
+    height: auto;
+    margin-top: 3rem;
+    margin-bottom: 70%;
+  }
+
+  /** latest-deals-ipad **/
+  .latest-deals-container {
+    display: -ms-grid;
+    display: grid;
+    grid-template: 1fr 1fr 1fr / repeat(3, 33%);
+    grid-column-gap: 2%;
+    grid-row-gap: 2%;
+    width: 100%;
+    height: auto;
+    padding-bottom: 5%;
+  }
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
- [ ] Not only is this media query repeated on line 404, you don't need to state a breakpoint for it. Instead, for example, go to your browser inspect tool, toggle the device toolbar, and set the width to 767px. Then write CSS styles and rules that will directly work for this screen size with no media query. This will mean that you are working mobile-first.  Then apply the medium and large screen sizes with media queries.